src/broadcast/spectateManager.ts

```diff
@@ -158,6 +159,8 @@ export class SpectateManager extends EventEmitter {
             case "list-broadcasts-resp": {
               const broadcasts: BroadcasterItem[] = obj.broadcasts ?? [];
               this.emit(SpectateEvent.BROADCAST_LIST_UPDATE, broadcasts);
+              this.broadcastList = {};
+              broadcasts.forEach((broadcast) => (this.broadcastList[broadcast.id] = broadcast));
```

Comment:
```suggestion
              this.broadcastList = {};
              broadcasts.forEach((broadcast) => {
                this.broadcastList[broadcast.id] = broadcast;
              });
```
Can we have this in a curly brace?